docs/docs/operator/telemetry.md

```diff
@@ -1,18 +1,14 @@
 # Setting up Telemetry
 
-Several of Kubescape's in-cluster components implement telemetry data using [OpenTelemetry](https://opentelemetry.io/) (otel).
-You can optionally install an otel [collector](https://opentelemetry.io/docs/collector/) to your cluster to aggregate all metrics and send them to your own tracing tool.
+Several of Kubescape's in-cluster components implement telemetry data using [OpenTelemetry](https://opentelemetry.io/) (otel).\
+You can optionally install an Otel [collector](https://opentelemetry.io/docs/collector/) to your cluster to aggregate all metrics and send them to your own tracing tool.
 
-You simply have to fill in this information before [installing kubescape operator](#installing-kubescape-operator-in-a-kubernetes-cluster-using-helm):
+You simply have to fill in this information before [installing kubescape operator](/../install-operator.md):\
+you need to edit the property below at [values.yaml](https://github.com/kubescape/helm-charts/blob/main/charts/kubescape-operator/values.yaml) OR using --set while installing the helm chart\
 
 ```
-otelCollector:
-  enabled: true
-  endpoint:
-    host: <ip or dns for your gRPC otel endpoint>
-    port: 4317
-    insecure: <whether your otel endpoint requires ssl>
-    headers: <optional - map of headers required by tracing tool>
+configurations:
+  otelUrl: # default is empty ,add yours otel URL here 
```

Comment:
fix the English here

Reply:
also we should still include the `endpoint` settings because it's required depending on which otel backend we're talking to

Reply:
what do you mean ?

in capabillities?

Reply:
this configuration:
# opentelemetry collector
otelCollector:
  name: otel-collector

  endpoint:
    insecure: true
    headers:
      uptrace-dsn: ""

  # -- enable/disable hostmetrics collection
  hostmetrics:
    enabled: true
    scrapeInterval: 30s

  image:
    repository: docker.io/otel/opentelemetry-collector
    tag: 0.97.0
    pullPolicy: IfNotPresent

  resources:
    requests:
      cpu: 100m
      memory: 500Mi
    limits:
      cpu: 1
      memory: 1Gi

Reply:
just the `endpoint`:
```
endpoint:
  insecure: true
  headers:
  uptrace-dsn: ""
```

Reply:
the user is configuring this?

Reply:
he might need to, depending on which Otel endpoint he's using, for instance with uptrace https://uptrace.dev/get/get-started.html#resource-attributes

Reply:
so it is enough?
or to add

Reply:
the uptrace is all the configuration from this page?

Reply:
just merge it like that, it's fine